OptionSelector/DetailCategoryTableViewController.m

```diff
@@ -0,0 +1,55 @@
+//
+//  DetailCategoryTableViewController.m
+//  OptionSelector
+//
+//  Created by Jason Wang on 8/15/15.
+//  Copyright (c) 2015 Mike Kavouras. All rights reserved.
+//
+
+#import "DetailCategoryTableViewController.h"
+
+@interface DetailCategoryTableViewController ()
+
+@end
+
+@implementation DetailCategoryTableViewController
+
+- (void)viewDidLoad {
+    [super viewDidLoad];
+
+}
+
+
+#pragma mark - Table view data source
+
+- (NSInteger)numberOfSectionsInTableView:(UITableView *)tableView {
+    return 1;
+}
+
+- (NSInteger)tableView:(UITableView *)tableView numberOfRowsInSection:(NSInteger)section {
+    return self.category.options.count;
+}
+
+- (UITableViewCell *)tableView:(UITableView *)tableView cellForRowAtIndexPath:(NSIndexPath *)indexPath {
+    UITableViewCell *cell = [tableView dequeueReusableCellWithIdentifier:@"DetailViewIdentifier" forIndexPath:indexPath];
+    
+    NSString *categorySelected = self.category.options[indexPath.row];
+    cell.textLabel.text = categorySelected;
+    
+    if (self.category.options[indexPath.row] == self.category.selected) {
+        cell.accessoryType = UITableViewCellAccessoryCheckmark;
+    } else {
+        cell.accessoryType = UITableViewCellAccessoryNone;
+    }
+    
+    return cell;
+}
+
+- (void)tableView:(UITableView *)tableView didDeselectRowAtIndexPath:(NSIndexPath *)indexPath {
```

Comment:
You may have noticed that an option doesn't appear as "selected" in your detail view until you select another option. That's because you're implementing did**Deselect**RowAtIndexPath: when you should be implementing did**Select**RowAtIndexPath: (see the distinction?). The former gets called whenever a row is **de**selected, whereas the latter gets called whenever a row is selected.